PyInstaller/building/utils.py

```diff
@@ -391,32 +391,54 @@ def _rmtree(path):
-        * A glob, such files will be recursively copied into the top-level
-          target directory. (This is usually *not* what you want.)
-        * A directory, such files will be recursively copied into a new
-          target subdirectory whose name is such directory's basename.
-          (This is usually what you want.)
-
-    :param workingdir: Optional argument, if datas contains relative paths then
-                       paths are relative to this directory and all relative
-                       paths are converted to absolute.
+    Convert the passed list of hook-style 2-tuples into a returned list of
+    `TOC`-style 2-tuples.
+
+    Elements of the passed list are 2-tuples `(source_dir_or_glob, target_dir)`.
+    Elements of the returned list are 2-tuples `(target_file, source_file)`.
+    For backwards compatibility, note that the order of elements in the former
+    are the reverse of the order of elements in the latter!
+
+    Parameters
+    ----------
+    binaries_or_datas : list
+        List of hook-style 2-tuples (e.g., the top-level `binaries` and `datas`
+        attributes defined by hooks) whose:
+        * First element is either:
+          * A glob matching only the absolute or relative paths of source
+            non-Python data files. The second element is then either:
+            * The relative path of the target directory into which these source
+              files will be recursively copied.
+            * The empty string, in which case these source files will be
+              recursively copied into the top-level target directory. (This is
+              usually _not_ what you want.)
+          * The absolute or relative path of a source directory containing only
+            source non-Python data files. The second element is then either:
+            * The relative path of the target directory into which these source
+              files will be recursively copied.
+            * The empty string, in which case these source files will be
+              recursively copied into a new target subdirectory whose name is
+              this source directory's basename. (This is usually what you want.)
+        If the optional `workingdir` parameter is passed, source paths may be
+        either absolute or relative; else, source paths _must_ be absolute.
+    workingdir : str
+        Optional absolute path of the directory to which all relative source
+        paths in the `binaries_or_datas` parameter will be prepended by (and
+        hence converted into absolute paths) _or_ `None` if these paths are to
+        be preserved as relative. Defaults to `None`.
+
+    Returns
+    ----------
+    list
+        List of `TOC`-style 2-tuples whose:
+        * First element is the absolute or relative path of a target file.
+        * Second element is the absolute or relative path of the corresponding
+          source file to be copied to this target file.
```

Comment:
Since we are aiming to convert the documentation to sphinx, I assume sphinx format is the only choice.